lib/inspec/utils/profile_ast_helpers.rb

```diff
@@ -3,8 +3,7 @@
 module Inspec
   class Profile
     class AstHelper
-      class CollectorBase
-        include Parser::AST::Processor::Mixin
+      class CollectorBase < Parser::AST::Processor
```

Comment:
I've verified that this applies to older versions (i.e. < 3.3.1.0) of `parser` gem  as well. I believe we don't need to explicitly specify the gem in the gemspec with that constraint. What do you think?

Reply:
Ah, right. Thanks.